manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
-# Title
+# Moving towards large-scale data-intensive forecasting of species richness in breeding birds
```

Comment:
I'm not happy with the title either. I'm not convinced that the current introduction quick backs up this alternative. I'd propose that we see how the paper shapes up and then tinker with the title.